### ISSUE TITLE
Should we decrease epochs if min_snr_gamma = 5?

### ISSUE THREAD

**Issue by teebarjunk** [Linaqruf/kohya-trainer#201]
This seems too good to be true, setting `min_snr_gamma = 5` has been giving really good sample images at the first epoch, and they only get better. The loss rate goes very low even with large diverse image sets that had higher loss rates when I've trained with them before.

Should we lower the `num_epochs` or `dataset_repeats`?
